ionoscloud/resource_ipfailover_test.go

```diff
@@ -198,24 +197,216 @@ resource "ionoscloud_server" "webserver" {
+  ram = 1024
+  availability_zone = "ZONE_1"
+  cpu_family = "AMD_OPTERON"
+  image_name = "ubuntu:latest"
+  image_password = ` + constant.RandomPassword + `.server_image_password.result
+  volume {
+    name = "system"
+    size = 15
+    disk_type = "SSD"
+  }
+  nic {
+    lan = "${ionoscloud_lan.webserver_lan1.id}"
+    dhcp = true
+    firewall_active = true
+     ips =["${ionoscloud_ipblock.webserver_ip.ips[0]}", "${ionoscloud_ipblock.webserver_ip.ips[1]}"]
+  }
+}
+
+resource "ionoscloud_server" "secondwebserver" {
+  depends_on = [ionoscloud_server.webserver]
+  name = "secondserver"
+  datacenter_id = "${ionoscloud_datacenter.foobar.id}"
+  cores = 1
+  ram = 1024
+  availability_zone = "ZONE_1"
+  cpu_family = "AMD_OPTERON"
+  image_name = "ubuntu:latest"
+  image_password = ` + constant.RandomPassword + `.server_image_password.result
+  volume {
+    name = "system"
+    size = 15
+    disk_type = "SSD"
+  }
+  nic {
+    lan = "${ionoscloud_lan.webserver_lan1.id}"
+    dhcp = true
+    firewall_active = true
+     ips =["${ionoscloud_ipblock.webserver_ip.ips[2]}"]
+  }
+}
+
+resource "` + constant.ResourceIpFailover + `" "` + constant.IpfailoverName + `" {
+  datacenter_id = "${ionoscloud_datacenter.foobar.id}"
+  lan_id="${ionoscloud_lan.webserver_lan1.id}"
+  ip ="${ionoscloud_ipblock.webserver_ip.ips[0]}"
+  nicuuid= "${ionoscloud_server.webserver.primary_nic}"
+}
+
+resource "` + constant.ResourceIpFailover + `" "` + constant.SecondIpfailoverName + `" {
+  depends_on = [ ` + constant.ResourceIpFailover + `.` + constant.IpfailoverName + ` ]
+  datacenter_id = "${ionoscloud_datacenter.foobar.id}"
+  lan_id="${ionoscloud_lan.webserver_lan1.id}"
+  ip ="${ionoscloud_ipblock.webserver_ip.ips[1]}"
+  nicuuid= "${ionoscloud_server.webserver.primary_nic}"
+}
+
+` + ServerImagePassword
+
+const testAccCheckLanIPFailoverGroupUpdateIp = `
+resource "ionoscloud_datacenter" "foobar" {
+	name       = "ipfailover-test"
+	location = "us/las"
+}
+
+resource "ionoscloud_ipblock" "webserver_ip" {
+  location = "us/las"
+  size = 3
+  name = "failover test"
+}
+
+resource "ionoscloud_lan" "webserver_lan1" {
+  datacenter_id = "${ionoscloud_datacenter.foobar.id}"
+  public = true
+  name = "terraform test"
+}
+
+resource "ionoscloud_server" "webserver" {
+  name = "server"
+  datacenter_id = "${ionoscloud_datacenter.foobar.id}"
+  cores = 1
+  ram = 1024
+  availability_zone = "ZONE_1"
+  cpu_family = "AMD_OPTERON"
+  image_name = "ubuntu:latest"
+  image_password = ` + constant.RandomPassword + `.server_image_password.result
+  volume {
+    name = "system"
+    size = 15
+    disk_type = "SSD"
+  }
+  nic {
+    lan = "${ionoscloud_lan.webserver_lan1.id}"
+    dhcp = true
+    firewall_active = true
+     ips =["${ionoscloud_ipblock.webserver_ip.ips[0]}"]
+  }
+}
+
+resource "ionoscloud_server" "secondwebserver" {
+  depends_on = [ionoscloud_server.webserver]
+  name = "secondserver"
+  datacenter_id = "${ionoscloud_datacenter.foobar.id}"
```

Comment:
do we really need ${} to get the id?

Reply:
@cristiGuranIonos ofc no, I kept it the same way because I reused the configuration, but I removed it in: ed1e170620e3d94cb90595dc3047a47258d4ae99